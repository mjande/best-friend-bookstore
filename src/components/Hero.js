import "../styles/Hero.css";

const Hero = () => {
  return (
    <section className="hero">
      <div className="hero-img"></div>
      <h2 className="hero-quote">
        “Outside of a dog, a book is man's best friend. Inside of a dog it's too
        dark to read.” <br /> Groucho Marx
      </h2>
    </section>
  );
};

export default Hero;
