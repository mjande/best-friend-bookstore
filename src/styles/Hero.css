.hero {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 38vh;
}

.hero-img {
  background-image: url("..//img/pexels-ksenia-chernaya-3952071.jpg");
  background-position: center;
  background-size: cover;
  position: absolute;
  filter: brightness(0.3);
  top: 12vh;
  z-index: -1;
  height: 38vh;
  width: 100vw;
}

q {
  color: white;
  text-align: center;
  width: 550px;
  font-size: 1.5rem;
}

p {
  color: white;
  margin: 0.5rem;
  margin-right: 1rem;
  font-size: 1.2rem;
  text-align: right;
}
