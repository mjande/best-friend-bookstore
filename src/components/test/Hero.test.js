import React from "react";
import { render } from "@testing-library/react";
import Hero from "../Hero";

describe("Hero", () => {
  it("renders hero without errors", () => {
    const { container } = render(<Hero />);
    expect(container).toMatchSnapshot();
  });
});
