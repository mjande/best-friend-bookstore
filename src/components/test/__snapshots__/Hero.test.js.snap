// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders hero without errors 1`] = `
<div>
  <section
    class="hero"
  >
    <div
      class="hero-img"
    />
    <h2
      class="hero-quote"
    >
      “Outside of a dog, a book is man's best friend. Inside of a dog it's too dark to read.” 
      <br />
       Groucho Marx
    </h2>
  </section>
</div>
`;
